packages/autocomplete-preset-algolia/src/requester/getAlgoliaResults.ts

```diff
@@ -0,0 +1,5 @@
+import { createAlgoliaRequester } from './createAlgoliaRequester';
+
+export const getAlgoliaResults = createAlgoliaRequester({
+  transformResponse: (result) => result.hits,
```

Comment:
```suggestion
  transformResponse: (response) => response.hits,
```

this seems the same as the other place I commented this on, are the files duplicate?

Reply:
It uses a different `createAlgoliaRequester`, the one in `autocomplete-js` passes user agents.

Reply:
See 6e42632d0310351822bb698a9000fe14f7acdcc1.

Reply:
@Haroenv I will probably refactor this in another PR where I need to add custom behavior before passing the search params. It will remove the duplicated code.